deploy/charts/cert-manager/values.yaml

```diff
@@ -416,6 +421,10 @@ hostAliases: []
 nodeSelector:
   kubernetes.io/os: linux
 
+# Property to set cert-manager-controller to run in pod namespace. To use host's properties set this value to true.
+# +docs:property
+# hostUsers: false
+
```

Comment:
I don't think we need to provide this feature per component. The global setting should be sufficient. IMO we should remove the global property and set `hostUsers: false` unconditionally when we can to improve the overall security of cert-manager.